src/api/api.ts

```diff
@@ -1 +1,16 @@
-export const BASE_URL = 'https://mate.academy/students-api';
+export const API_URL = 'https://mate.academy/students-api';
+
+export const request = (
+  endpoint: string,
+  options?: RequestInit | undefined,
```

Comment:
Думаю "undefined" зайвий
```suggestion
  options?: RequestInit,
```